app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -219,6 +219,28 @@ export const BranchesForm: React.FC = () => {
     dispatch(updateExperiment({ variants: updatedVariants }));
   };
 
+  const NEW_BRANCH = {
+    name: "",
+    is_control: false,
+    description: "An empty branch",
+    value: "",
+    ratio: 50,
```

Comment:
Although this produces the results we actually want (which is for new branches to be the same ratio by default ) the UI doesn't make sense in terms of its representation as a percentage

Reply:
would 0 be a better default value then?

Reply:
Ok, as discussed lets do the following for now:
- make this field disabled / not editable
- always set the value of ratio to `1` (including for new branches)
- in the UI, render it as a percentage to 2 decimal points
- remove the validator in the serializer requiring this to add up to 100%
- file a ticket to make ratio editable, which will involve doing client-side validation of adding to 100% and converting back to ratios from percentages (or maybe a different ui for ratios?)